.circleci/config.yml

```diff
@@ -137,6 +123,36 @@ jobs:
       - run:
           name: Run nox on speech
           <<: *anchor_test_python
+  test-ruby:
+    working_directory: /tmp/
+    docker:
+      - image: circleci/ruby:2.4-node
+    steps:
+      - attach_workspace:
+          # Use the workspace created in generate-clients
+          at: workspace
+      - run:
+          name: Install Ruby client dependencies.
+          command: |
+            gem install bundler
```

Comment:
Bundler is usually pre-installed on ruby docker images.

Reply:
great! removed